app/javascript/components/widgets/widgets/forest-change/fao-reforest/actions.js

```diff
@@ -8,7 +8,12 @@ export const getData = ({ params, dispatch, setWidgetData, widget, state }) => {
         (data.length &&
           data.find(d => d.iso === state().location.payload.country)) ||
         null;
-      dispatch(setWidgetData({ data: hasCountryData ? data : {}, widget }));
+      dispatch(
+        setWidgetData({
+          data: hasCountryData || params.type === 'global' ? data : {},
```

Comment:
Why not remove the country dependancy from the check above instead? So that you only check for data.length

Reply:
Good spot.